pkg/gui/view_helpers.go

```diff
@@ -217,7 +218,7 @@ func (gui *Gui) renderString(g *gocui.Gui, viewName, s string) error {
 			return nil
 		}
 		v.Clear()
-		fmt.Fprint(v, s)
+		fmt.Fprint(v, bom.Clean([]byte(s)))
```

Comment:
I'm only seeing an array of bytes here. Looks like if we're going to use fmt.Fprint() we need to give it a string rather than a byte slice
![image](https://user-images.githubusercontent.com/8456633/44305843-cabbf300-a3c5-11e8-9517-5a44985be798.png)

Looks like something like this would do the trick
```
		buffer := bytes.NewBuffer(bom.Clean([]byte(s)))
		buffer.WriteTo(v)
```
Though when I did that locally it reverted to just showing the last line of the CSV again

Reply:
Closing this for now until I figure out a fix